src/Native/Runtime/amd64/PInvoke.asm

```diff
@@ -341,4 +340,59 @@ LEAF_ENTRY RhpReversePInvokeReturn, _TEXT
 LEAF_END RhpReversePInvokeReturn, _TEXT
 
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; RhpPInvoke
```

Comment:
This should probably have some comments about the invariants expected by this simple helper.  i.e. no live GC refs in preserved registers, etc..